tensorboard/main.py

```diff
@@ -41,7 +42,11 @@ def main(unused_argv=None):
 
   See `tensorboard.program.main` for further documentation.
   """
-  return program.main(default.get_plugins(),
+  external_plugins = [
```

Comment:
Hmm, are you sure it isn't easier to just filter out BeholderPlugin for the places that might not want to include it?  Having a separate sort of quasi-default list of plugins here - that are by default enabled in TensorBoard release versions, but not in default.get_plugins() - seems pretty weird to me.